tests/integration/test_app.py

```diff
@@ -20,40 +20,36 @@
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
 
-from __future__ import absolute_import, division, print_function
-
 import json
+from math import isclose
 
 
 def test_health_check(app_client):
     assert app_client.get('/api/health').status_code == 200
 
 
-def test_classifier_accepts_only_post(app_client):
+def test_classifier_accepts_only_post(app, app_client, trained_pipeline):
     assert app_client.post('/api/classifier', data=dict(title='foo bar', abstract='foobar foobar')).status_code == 200
     assert app_client.get('/api/classifier').status_code == 405
 
 
-def test_classifier(app_client):
+def test_classifier(app, app_client, trained_pipeline):
     response = app_client.post('/api/classifier', data=dict(title='foo bar', abstract='foobar foobar'))
 
     result = json.loads(response.data)
 
-    expected = {
-        "prediction": "core",
-        "score1": 0.1,
-        "score2": 0.2,
-        "score3": 0.7,
-    }
-
     assert response.status_code == 200
-    assert expected == result
+    assert len(result) == 2
+    assert set(result.keys()) == {'prediction', 'scores'}
```

Comment:
isn't the assertion on the length redundant if you assert on the set of keys? On the other hand, you could assert that
`result['prediction'] in {'rejected', 'non_core', 'core'}`

Reply:
Yeah, it's kind of redundant but I had it there since we were taking a set later on.

Reply:
The suggested assertion is good though. I'll add it.

Reply:
Wait, never mind the comment about the set. That's incorrect.